src/AppGui.cpp

```diff
@@ -89,8 +89,12 @@ bool AppGui::initialize()
     QMenu *systrayMenu = new QMenu();
 
     daemonAction = new DaemonMenuAction(systrayMenu);
-    systrayMenu->addAction(daemonAction);
-    systrayMenu->addSeparator();
+    // FIXME: Shall we add a fallback for linux and MacOs?
+    #ifdef Q_OS_WIN
```

Comment:
No. This works on macos, it's only on linux that does not support widgets in QMenu

Reply:
Also on linux, normal qmenu entries could be added for starting/stopping the daemon